### PR TITLE
Display default values for EnvironmentOptions in katana cli

### LINKED ISSUE #1815: [katana] Display missing default values in command line --help
**Is your feature request related to a problem? Please describe.**
The following default values are not displayed : 
```
--validate-max-steps <VALIDATE_MAX_STEPS>
The maximum number of steps available for the account validation logic.

 --invoke-max-steps <INVOKE_MAX_STEPS>
The maximum number of steps available for the account execution logic.

--eth-gas-price <L1_ETH_GAS_PRICE>
The L1 ETH gas price.

--strk-gas-price <L1_STRK_GAS_PRICE>
The L1 STRK gas price.
```

**Describe the solution you'd like**
Display default values like other params
```
-p, --port <PORT>
Port number to listen on.

[default: 5050]
```

### PR DESCRIPTION
**Pull request by notV4l**
fixes #1815 

add extra logs at start

```
ENVIRONMENT
==================

| Chain ID            | 0x4b4154414e41
| Disable Fee         | false
| Disable Validate    | false
| Validate max steps  | 1000000
| Invoke max steps    | 1000000
| ETH Gas Price       | 1112701192
| STRK Gas Price      | 0

FORKING
==================
    
| Fork rpc url        | https://api.cartridge.gg/rpc/starknet
| Fork blocknumber    | 42

```

### CODE REVIEW

File: bin/katana/src/args.rs [hunk 1]
```diff
@@ -175,21 +175,25 @@ pub struct EnvironmentOptions {
 
     #[arg(long)]
     #[arg(help = "The maximum number of steps available for the account validation logic.")]
-    pub validate_max_steps: Option<u32>,
+    #[arg(default_value = format!("{:?}",DEFAULT_VALIDATE_MAX_STEPS))]
```
Comment (kariy) [inline bin/katana/src/args.rs:178]:
```suggestion
    #[arg(default_value_t = DEFAULT_VALIDATE_MAX_STEPS)]
```

File: bin/katana/src/args.rs [hunk 2]
```diff
@@ -175,21 +175,25 @@ pub struct EnvironmentOptions {
...
+    pub validate_max_steps: u32,
 
     #[arg(long)]
     #[arg(help = "The maximum number of steps available for the account execution logic.")]
-    pub invoke_max_steps: Option<u32>,
+    #[arg(default_value = format!("{:?}",DEFAULT_INVOKE_MAX_STEPS))]
```
Comment (kariy) [inline bin/katana/src/args.rs:183]:
```suggestion
    #[arg(default_value_t = DEFAULT_INVOKE_MAX_STEPS)]
```

File: bin/katana/src/args.rs [hunk 3]
```diff
@@ -175,21 +175,25 @@ pub struct EnvironmentOptions {
...
+    pub invoke_max_steps: u32,
 
     #[arg(long = "eth-gas-price")]
     #[arg(conflicts_with = "genesis")]
-    #[arg(help = "The L1 ETH gas price.")]
-    pub l1_eth_gas_price: Option<u128>,
+    #[arg(help = "The L1 ETH gas price. (denominated in wei)")]
+    #[arg(default_value = format!("{:?}",DEFAULT_ETH_L1_GAS_PRICE))]
```
Comment (kariy) [inline bin/katana/src/args.rs:189]:
```suggestion
    #[arg(default_value_t = DEFAULT_ETH_L1_GAS_PRICE)]
```

File: bin/katana/src/args.rs [hunk 4]
```diff
@@ -175,21 +175,25 @@ pub struct EnvironmentOptions {
...
+    pub l1_eth_gas_price: u128,
 
     #[arg(long = "strk-gas-price")]
     #[arg(conflicts_with = "genesis")]
-    #[arg(help = "The L1 STRK gas price.")]
-    pub l1_strk_gas_price: Option<u128>,
+    #[arg(help = "The L1 STRK gas price. (denominated in fri)")]
+    #[arg(default_value = format!("{:?}",DEFAULT_STRK_L1_GAS_PRICE))]
```
Comment (kariy) [inline bin/katana/src/args.rs:195]:
```suggestion
    #[arg(default_value_t = DEFAULT_STRK_L1_GAS_PRICE)]
```

File: Cargo.toml [hunk 5]
```diff
@@ -127,7 +127,7 @@ cairo-proof-parser = { git = "https://github.com/cartridge-gg/cairo-proof-parser
 cairo-vm = "0.9.2"
 camino = { version = "1.1.2", features = [ "serde1" ] }
 chrono = { version = "0.4.24", features = [ "serde" ] }
-clap = { version = "4.5", features = [ "derive" ] }
+clap = { version = "4.5", features = [ "derive", "string" ] }
```
Comment (kariy) [inline Cargo.toml:130]:
```suggestion
clap = { version = "4.5", features = [ "derive" ] }
```

**Review by kariy — CHANGES_REQUESTED** [review]
im not sure about the prints. i have some idea about displaying the config on startup and the config types are a bit messy right now, so would rather so would prefer if we dont print anything else for now.

File: bin/katana/src/args.rs (hunk 1 above)
Comment (notV4l) [reply bin/katana/src/args.rs:178]: weird i remind i tried with default_value_t

**Comment by notV4l** [comment]
> im not sure about the prints. i have some idea about displaying the config on startup and the config types are a bit messy right now, so would rather so would prefer if we dont print anything else for now.

idk why its an issue to diplay more infos but up to you

**Review by kariy — APPROVED** [review]
lgtm

**Pull request merged**